doc/contribute_models.md

```diff
@@ -97,7 +89,54 @@ class MyAwesomeClassifier(tf.keras.Model):
         # do custom training here, parameter "dataset" is a tf.dataset type representing the input data.
 ```
 
-In `__init__.py` you should expose your model classes by adding lines like `from mydnnclassifier import MyAwesomeClassifier`. After this, you can test your model following the below step.
+### Export Model Definition
+
+In `__init__.py` you should expose your model classes by adding lines like:
+
+```python
+from awesome_model import MyAwesomeClassifier
+```
+
+### Write the Dockerfile
+
+To build a Docker image, we need a file named `Dockerfile` like below:
+
+```dockerfile
+# base image sqlflow/modelzoo_base have SQLFlow environment prepared
+FROM sqlflow/modelzoo_base
+RUN pip install scikit-learn six
+# copy model definition code to /sqlflow_models
+ADD your_model_package /sqlflow_models
```

Comment:
I don't see the `your_model_package` directory in the above text. Do you mean `my_model_zoo`?